CHANGELOG.md

```diff
@@ -5,10 +5,48 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.11.0] - 2022-11-07
+
+### Added
+- Storage encryption functionality in `irmaclient`
+- Challenge response user authentication using ECDSA key pair between `irma keyshare server` and `irmaclient`
```

Comment:
Maybe add a note that there is a breaking change: `irmaclient` cannot talk against `irma keyshare server` < 0.11.0.

Reply:
Done.